### ISSUE TITLE
Upgrade to 5.0.0 breaks on non-windows machines

### ISSUE THREAD

**Issue by deviantintegral** [tbroadley/spellchecker-cli#95]
It looks to be exactly the same issue as https://github.com/tbroadley/spellchecker-cli/issues/86:

```console
$ ./node_modules/.bin/spellchecker
env: node\r: No such file or directory
```

**Comment by tbroadley** [comment]
Thank you! I just published 5.0.1 to fix this, and updated RELEASE.md so hopefully I don't make this mistake again.

**Comment by deviantintegral** [comment]
Looks good to me! Thank you.
